docs/pages/inboxes/build-inbox.md

```diff
@@ -0,0 +1,172 @@
+## Build the inbox
+
+:::warning[🚧 XMTP V3 Alpha]
+
+The XMTP V3 SDKs are in **alpha** status and ready for you to start experimenting with.
+However, we do **not** recommend using alpha software in production apps. Software in this status will change as we add features and iterate based on feedback.
```

Comment:
Should we / do we really need to say this with Converse already using v3 in production?